ddtrace/internal/writer.py

```diff
@@ -555,11 +555,12 @@ def write(self, spans=None):
         except BufferFull as e:
             payload_size = e.args[0]
             log.warning(
-                "trace buffer (%s traces %db/%db) cannot fit trace of size %db, dropping",
+                "trace buffer (%s traces %db/%db) cannot fit trace of size %db, dropping. writer status: %s",
```

Comment:
```suggestion
                "trace buffer (%s traces %db/%db) cannot fit trace of size %db, dropping (writer status: %s)",
```
or capital `Writer`?

Reply:
🤷🏻 no strong preference. what do you like?

Reply:
I like the current code suggestion with the `(...)` more 🍭

Reply:
done.